.pre-commit-config.yaml

```diff
@@ -53,7 +53,7 @@ repos:
 
   # Spell check our documentation and code comments
   - repo: https://github.com/crate-ci/typos
-    rev: v1.24.5
+    rev: v1.34.0
```

Comment:
For reasons I don't understand, `pre-commit autoupdate` tried to change this to `v1` rather than a specific version, which `pre-commit` does not support -- the ref has to point at a single, stable version.  Updating manually but not sure what will happen in the future.